server/server.js

```diff
@@ -25,27 +25,45 @@ app.use(
         'default-src': ["'self'"],
         'connect-src': [
           "'self'",
-          'https://*.apps.gov.bc.ca',
```

Comment:
hmmm do we have a sense of what this was originally added to support? might be worth a git-blame and checking the PR

Reply:
I check line history this was added as part of initial helmet config.